test/scaladoc/run/tag-requirements.check

```diff
@@ -0,0 +1,16 @@
+newSource:3: warning: Only one '@version' tag is allowed
+        /**
+        ^
+newSource:9: warning: Tag '@param' must be followed by a symbol name
+          /**
+          ^
+newSource:9: warning: Tag '@param' is not recognised
```

Comment:
I think we could raise a later improvement which removes this warning on the basis that the `param` tag has already been warned on. It's not actually unrecognised but unused because of the missing symbol name.